common/model/exhibitions.js

```diff
@@ -46,6 +46,8 @@ export type ExhibitionPromo = {|
   url: string,
   title: string,
   image: Image,
+  squareImage: ?Picture;
+  thinImage: ?Picture;
```

Comment:
We've not added these to the [`GenericContentTypes`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/model/generic-content-fields.js).
As it's something we'll be using else where, could we add it there and to the [generic parsers](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/parsers.js#L506)?